### PR TITLE
env_optimization

### PR DESCRIPTION
**Pull request by sumanvpacewisdom**

### CODE REVIEW

File: src/envVariables.js [hunk 1]
```diff
@@ -41,89 +49,105 @@ let enviromentVariables = {
...
 		message: 'Redis Host Url',
 		optional: false,
 	},
 	KEY: {
 		message: 'Key is missing for email encryption',
-		optional: false,
+		optional: true,
+		default: 'E/m3RD/aM3Ed3lLfYVcKizakG9R+bFybAPZSLjIP2hY=',
 	},
 	IV: {
```
Comment (nevil-mathew) [inline src/envVariables.js:94]: Are these used anywhere in the code?

File: src/envVariables.js [hunk 2]
```diff
@@ -41,89 +49,105 @@ let enviromentVariables = {
...
 		message: 'iv is missing for email encryption',
-		optional: false,
+		optional: true,
+		default: 'LHYOA5YnTonqcgrm15k3/Q==',
```
Comment (nevil-mathew) [inline src/envVariables.js:97]: Are these used anywhere in the code? If not, please remove this.

File: src/envVariables.js [hunk 3]
```diff
@@ -41,89 +49,105 @@ let enviromentVariables = {
...
+		default: 'invite_org_admin',
 	},
 	DEFAULT_ORG_ID: {
 		message: 'Default organization ID, Run seeder `insertDefaultOrg` to obtain it.',
-		optional: false,
+		optional: true,
+		default: 1,
 	},
 	PORTAL_URL: {
 		message: 'Required portal url',
```
Comment (nevil-mathew) [inline src/envVariables.js:153]: Please check if this is used anywhere, else remove it.

File: src/envVariables.js [hunk 4]
```diff
@@ -135,33 +159,47 @@ let enviromentVariables = {
...
-		optional: false,
+		optional: true,
+		default: '9bbf4698e081176b4e38c3f04f9c0cff6ae80b1960e5ac868cc234ac8537aa66',
 	},
 	EMAIL_ID_ENCRYPTION_IV: {
 		message: 'Required Email ID Encryption IV',
-		optional: false,
+		optional: true,
+		default: 'a390328e496f025249a88384f318a536',
 	},
```
Comment (nevil-mathew) [inline src/envVariables.js:179]: These cannot be optional, please remove the default value as well.

File: src/envVariables.js [hunk 5]
```diff
@@ -135,33 +159,47 @@ let enviromentVariables = {
...
 	EMAIL_ID_ENCRYPTION_ALGORITHM: {
 		message: 'Required Email ID Encryption Algorithm',
-		optional: false,
+		optional: true,
 		default: 'aes-256-cbc',
 	},
+	ENABLE_EMAIL_OTP_VERIFICATION: {
+		message: 'Required Email otp verification ',
+		optional: true,
+		default: 'false',
```
Comment (nevil-mathew) [inline src/envVariables.js:188]: Please update the default value to be true.

**Review by nevil-mathew — CHANGES_REQUESTED** [review]

File: src/envVariables.js [hunk 6]
```diff
@@ -176,48 +214,55 @@ let enviromentVariables = {
...
-		default: 0,
+		default: 3600,
 	},
 	CHANGE_PASSWORD_TEMPLATE_CODE: {
 		message: 'Required change password email template code',
-		optional: false,
+		optional: true,
+		default: 'change_password',
 	},
 	CAPTCHA_ENABLE: {
```
Comment (nevil-mathew) [inline src/envVariables.js:240]: add the default value as false

File: src/envVariables.js [hunk 7]
```diff
@@ -176,48 +214,55 @@ let enviromentVariables = {
...
 	GOOGLE_RECAPTCHA_HOST: {
 		message: 'Required CAPTCHA Host IP',
-		optional: true,
+		optional: process.env.CAPTCHA_ENABLE === 'false' ? true : false,
 		default: 'https://www.google.com',
 	},
 	GOOGLE_RECAPTCHA_URL: {
 		message: 'Required CAPTCHA SERVICE API URL',
-		optional: true,
+		optional: process.env.CAPTCHA_ENABLE === 'false' ? true : false,
```
Comment (nevil-mathew) [inline src/envVariables.js:265]: Make this as optional true

File: src/envVariables.js (hunk 3 above)
Comment (sumanvpacewisdom) [reply src/envVariables.js:153]: it is used in 4 places

**Pull request merged**